iceberg/iceberg-common/src/test/java/org/apache/gravitino/iceberg/common/utils/TestIcebergCatalogUtil.java

```diff
@@ -78,4 +78,52 @@ void testLoadCatalog() {
           IcebergCatalogUtil.loadCatalogBackend("other");
         });
   }
+
+  @Test
+  void testValidLoadCustomCatalog() {
+    Catalog catalog;
+    Map<String, String> config = new HashMap<>();
+
+    config.put("catalog-backend-impl", "org.apache.iceberg.inmemory.InMemoryCatalog");
+    catalog =
+        IcebergCatalogUtil.loadCatalogBackend(
+            IcebergCatalogBackend.valueOf("CUSTOM"), new IcebergConfig(config));
+    Assertions.assertTrue(catalog instanceof InMemoryCatalog);
+
+    config.clear();
+    config.put("catalog-backend-impl", "org.apache.iceberg.hive.HiveCatalog");
+    catalog =
+        IcebergCatalogUtil.loadCatalogBackend(
+            IcebergCatalogBackend.valueOf("CUSTOM"), new IcebergConfig(config));
+    Assertions.assertTrue(catalog instanceof HiveCatalog);
+
+    config.clear();
+    config.put(
+        "catalog-backend-impl", "org.apache.gravitino.iceberg.common.utils.CustomCatalogForTest");
```

Comment:
seems use `org.apache.gravitino.iceberg.common.utils.CustomCatalogForTest` is enough, no need to test `memory` or `hive`?

Reply:
ok, I'll remove it